static/js/settings_data.js

```diff
@@ -158,3 +158,18 @@ export function user_can_edit_topic_of_any_message() {
     }
     return user_has_permission(page_params.realm_edit_topic_policy);
 }
+
+export function using_dark_theme() {
```

Comment:
Any other suggestions for the name of this function?

Reply:
I think that function name is pretty reasonable.